### PR TITLE
acts: new versions 19.4.0 and 19.5.0

### PR DESCRIPTION
**Pull request by wdconinc**
Two new versions of `acts`, minor build system changes in https://github.com/acts-project/acts/compare/v19.4.0...v19.5.0 and https://github.com/acts-project/acts/compare/v19.3.0...v19.4.0.

### CODE REVIEW

**Comment by wdconinc** [comment]
@HadrienG2 Leaving this still as a draft since I don't know exactly what the philosophy behind the log threshold changes is and whether we should duplicate the variants too.

**Review by alalazo — APPROVED** [review]
Waiting for maintainers approval before merging

**Review by vvolkl — APPROVED** [review]

**Pull request merged**
